cmd/demo/node.go

```diff
@@ -25,23 +25,25 @@ import (
 	"perun.network/go-perun/channel"
 	"perun.network/go-perun/client"
 	"perun.network/go-perun/log"
-	perunpeer "perun.network/go-perun/peer"
-	"perun.network/go-perun/peer/net"
 	"perun.network/go-perun/wallet"
+	"perun.network/go-perun/wire"
+	wirenet "perun.network/go-perun/wire/net"
+	wirenetsimple "perun.network/go-perun/wire/net/simple"
```

Comment:
```suggestion
	simple "perun.network/go-perun/wire/net/simple"
```
We only use names like `ethchannel` sometimes because there are multiple `channel`. For the `simple` there is only one so it should be fine just naming it `simple`.

Reply:
ok, the reasoning was that "simple" is a bit of a too simple name to be understood what this refers to when out of context. but i guess the context is given and, as you suggest, only using package aliases when there is a package name overload seems to be the standard way to do it.